certified-connectors/Telnyx SMS/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Telnyx"

```

Comment:
@fabianp5060 the stackOwner property is missing, please refer to the [apiProperties.json template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). The current iconBrandColor is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

Reply:
updated in commit [a415100](https://github.com/microsoft/PowerPlatformConnectors/pull/1711/commits/a415100a41164af6aff5ec47643e12a8f844a2b1)